libs/data-mapper-v2/src/core/state/DataMapSlice.ts

```diff
@@ -1,5 +1,6 @@
 import type { ConnectionDictionary, InputConnection } from '../../models/Connection';
 import { directAccessPseudoFunctionKey, type FunctionData, type FunctionDictionary } from '../../models/Function';
+import type { Draft } from 'immer';
```

Comment:
Temporary hack for build to pass: we have to import immer somewhere https://github.com/reduxjs/redux-toolkit/issues/1806

Reply:
(I removed immer in the below file)